src/SFA.DAS.ApprenticeFeedback.Jobs/Startup.cs

```diff
@@ -93,6 +97,11 @@ public override void Configure(IFunctionsHostBuilder builder)
                 .AddHttpMessageHandler<Http.MessageHandlers.DefaultHeadersHandler>()
                 .AddHttpMessageHandler<Http.MessageHandlers.ApimHeadersHandler>()
                 .AddHttpMessageHandler<Http.MessageHandlers.LoggingMessageHandler>();
+
+            builder.Services.AddRestEaseClient<IApprenticeFeedbackEmailTransactionApi>(url)
```

Comment:
This shouldn't be needed due to the other Outer Api already existing.

Reply:
Removed